lib/pages/app_grid/app_grid_dashboard_item_style.dart

```diff
@@ -1,10 +1,22 @@
-import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 
 class AppGridDashboardItemStyle {
-  static const double iconAppSize = 56.0;
-  static const double itemWidth = 98.0;
-  static const double borderRadius = 16.0;
+  static const double iconAppSize = 42.0;
+  static const double itemWidth = 80.0;
 
-  static const EdgeInsets padding = EdgeInsets.symmetric(vertical: 8);
+  static const EdgeInsets padding = EdgeInsets.only(
+    bottom: 4,
+    top: 8,
+  );
   static const EdgeInsets textPadding = EdgeInsets.only(top: 8);
+
+  static const BorderRadius borderRadius =
+      BorderRadius.all(Radius.circular(10));
+
+  static TextStyle? textStyle(BuildContext context) =>
+      Theme.of(context).textTheme.bodySmall?.copyWith(
+            fontSize: 12,
+            fontWeight: FontWeight.w400,
+            color: Colors.black,
+          );
```

Comment:
- Why need to use `foneSize` and `fontWeight`?
- Should use the color of `Linagora system design`

Reply:
Removed cause no need